src/controller/data-manager.js

```diff
@@ -191,10 +195,22 @@ class DataManager {
   fetchData() {
     // console.log('\nFETCH DATA');
     // console.log('-----------');
+    if(typeof this.store.state.activeCondo != 'undefined' && this.store.state.activeCondo.featureId != null) {
+
+      const geocodeObj = this.store.state.condoUnits.units[this.store.state.activeCondo.featureId];
+      const ownerSearchObj = this.store.state.condoUnits.units[this.store.state.activeCondo.featureId];
```

Comment:
Should both geocodeObj and ownerSearchObj be set to the same value?

Reply:
I suppose so since they are const and won't change.  

"const geocodeObj =ownerSearchObj = " is how you normally do this right? 
(There's no other simple one-line alternative?)